### PR TITLE
Add responsive menu

### PR DESCRIPTION
**Pull request by lkulbacki**
https://projects.kodilla.com/browse/WDP181201-11

### CODE REVIEW

File: src/scripts/app.js [hunk 1]
```diff
@@ -1 +1 @@
-console.log('hello');
\ No newline at end of file
+console.log('hello');
```
[the remark this answers is not in the document]
Comment (lkulbacki) [reply src/scripts/app.js]: Removed

File: src/sass/components/_header.scss [hunk 2]
```diff
@@ -291,3 +298,134 @@ header {
...
+      }
+      .menu {
+        order: 2;
+      }
+    }
+  }
+}
+
+@include media-breakpoint-up(md) {
+header {
```
[the remark this answers is not in the document]
Comment (lkulbacki) [reply src/sass/components/_header.scss]: Indented.

File: src/sass/components/_header.scss [hunk 3]
```diff
@@ -380,7 +380,7 @@ header {
     }
 
     .menu-bar {
```
Comment (pawel-schmidt) [inline src/sass/components/_header.scss]: Remove empty selector.

Comment (lkulbacki) [reply src/sass/components/_header.scss]: Done.

**Comment by pawel-schmidt** [comment]
@lkulbacki please resolve merge conflicts.

**Comment by lkulbacki** [comment]
@pawel-schmidt Merge conflicts resolved

**Pull request merged**